packages/editor-ui/src/composables/useCloudPlanHelper.ts

```diff
@@ -0,0 +1,8 @@
+import type { Cloud } from '@/Interface';
+
+export function useCloudPlanHelper() {
+	return {
+		userIsTrialing: (metadata: Cloud.PlanMetadata) =>
+			metadata?.trial && metadata?.group === 'opt-in',
```

Comment:
why `opt-in`?? the group is `trial`.. https://linear.app/n8n/issue/ADO-633#comment-46729ff5

Reply:
AHHHH, I saw the description and not the pictures. Now it makes sense. I will update the type to reflect trial as an option to the group